packages/react-devtools-shared/src/devtools/views/Profiler/SnapshotCommitList.js

```diff
@@ -186,22 +194,37 @@ function List({
       selectedCommitIndex,
       selectedFilteredCommitIndex,
       selectCommitIndex,
+      setHoveredCommitIndex,
     ],
   );
 
+  let tooltipLabel = null;
+  if (hoveredCommitIndex !== null) {
+    const commitDuration = commitDurations[hoveredCommitIndex];
+    const commitTime = commitTimes[hoveredCommitIndex];
+    tooltipLabel = `${formatDuration(commitDuration)}ms at ${formatTime(
+      commitTime,
+    )}s`;
```

Comment:
I pan to build on top of this in a future PR that adds the concept of commit times and nested update flags directly to the Profiler UI. Those will be added to this tooltip as well.